docs/guides/CustomStartupGuide.md

```diff
@@ -1,4 +1,4 @@
-# USB Camera Device Service RTSP Streaming Guide
+# USB Camera Device Service Custom Startup Guide
 
 ## Contents
```

Comment:
Make sure the sections these link to still exist. I think "How it works" and "License" aren't in this doc